plugins/flytekit-dbt/tests/test_task.py

```diff
@@ -0,0 +1,149 @@
+import os
+import pytest
+import subprocess
+
+from flytekit import workflow
+
+from flytekitplugins.dbt.error import DBTUnhandledError
+from flytekitplugins.dbt.schema import (
+    DBTRunInput,
+    DBTRunOutput,
+    DBTTestInput,
+    DBTTestOutput,
+)
+from flytekitplugins.dbt.task import DBTRun, DBTTest
+
+
+@pytest.fixture(scope="module", autouse=True)
+def prepare_db():
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    setup_db_script = os.path.join(dir_path, "..", "..", "flytekit-dbt", "tests", "setup_db.sh")
+
+    print("preparing db")
+    subprocess.run(setup_db_script)
+    print("db is ready")
+
+    yield
```

Comment:
Need your input on this part: setting up the integration test environment. In Gojek, our flytekit-dbt transformation test is targeting our BQ dataset which cannot be reproduced in open-source space. So, a more suitable choice is to use PostgreSQL. This approach assumes that PostgreSQL is already running - whether via binary or container.

Another approach that I've considered:
1. Mark these tests as integration tests, entirely separated from unit tests. It doesn't break existing tests but introduces more complexity on how to do testing. But if in the future there's another plugin that needs an integration test, it can inherit this setup

Reply:
how about sqlite3?

Reply:
* How about sqlite3?
Currently, sqlite is not officially supported by DBT. There's a community SQL adapter plugin (https://github.com/codeforkjeff/dbt-sqlite), but using it for integration testing is a different scenario than using the official adapter, we are ended up testing the adapter too.

* All the csv files are for integration testing right?
These CSV files are only for integration testing.

Reply:
Can we mark the integration tests as skipped for now?  We're going to be soon revisiting integration testing in flytekit in general.  Maybe in late April.  What do you think @eapolinario?

Reply:
mark as skipped and then comment out the setup step for now.

Reply:
Setting up postgres on the github CI boxes is pretty quick (the [initial run](https://github.com/flyteorg/flytekit/runs/5490579965?check_suite_focus=true#step:3:1) for this took 3s), so we should be ok leaving that enabled assuming that (1) we can make this work with postgres, and (2) the tests themselves are not flaky.